src/alexandria/protocol.rs

```diff
@@ -163,4 +161,16 @@ impl AlexandriaProtocol {
 
         Ok(response)
     }
+
+    /// Convenience call for testing, quick way to ping bootnodes
+    pub async fn ping_bootnodes(&mut self) -> Result<(), String> {
+        // trigger bonding with bootnodes
+        self.discovery.discover_nodes().await?;
```

Comment:
My understanding of `discover_nodes()` is that it pings random node ids to build the table, not any special hardcoded / cli-input "bootnode". Am I misunderstading? Maybe it's just semantic technicality and in practice any randomly pinged node id is in effect acting as a bootnode.

Reply:
Yeah, this definitely would be better with an explanation. Discovery pings the bootnodes, because the bootnodes are the only ENRs available to ask. That's the only way it can try to discover the random node ID that becomes the target. Importantly, it doesn't bond with all bootnodes, just (at least) one of them.

I kind of don't expect this method to be around for long (famous last words), but it's still worth adding a comment for sure.

Reply:
Actually, the even simpler answer is that it's totally unnecessary, because if we do the portal ping over `TALKREQ`, that will automatically trigger a session on the base layer.